renderer/webtorrent.js

```diff
@@ -63,6 +63,9 @@ function init () {
 
   ipc.send('ipcReadyWebTorrent')
 
+  window.addEventListener('error', (err) =>
```

Comment:
This event listener really calls your function with an event object, not an error, so naming it `e` would be clearer.